lib/constants.ts

```diff
@@ -40,3 +40,7 @@ export const YOUNG_ACCOUNT_MARKER_THRESHOLD_IN_DAYS = process.env
   .NEXT_PUBLIC_YOUNG_ACCOUNT_MARKER_THRESHOLD_IN_DAYS
   ? parseInt(process.env.NEXT_PUBLIC_YOUNG_ACCOUNT_MARKER_THRESHOLD_IN_DAYS)
   : 30
+
+export const DOMAINS_ALLOWING_EMAIL_COMMUNICATION = (
+  process.env.NEXT_PUBLIC_DOMAINS_ALLOWING_EMAIL_COMMUNICATION || 'bsky.network'
```

Comment:
Perhaps we should default to empty, and configure this specially on our deployments— otherwise this will be the configuration in the distro.